scripts/actions/spells/black/bio_iii.lua

```diff
@@ -45,20 +45,31 @@ spellObject.onSpellCast = function(caster, target, spell)
     -- http://wiki.ffo.jp/html/1954.html
     -- this is a tiered calculation that has at least three tiers,
     -- so I'll use breakpoints for human readability
-    local dotdmg = 0
-    if     skillLvl > 400 then dotdmg = 17
```

Comment:
ya know, I even called out this exact styling in the style guide years ago. _Reeeeeeee no padding between the if and the condition_

Ty for fixing.